src/OrefAlertsRetrieve.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import warnings
+warnings.filterwarnings('ignore')
+import requests
+import json
+from datetime import datetime, timedelta
+import time 
+from Errors import RetrieveDataException
+URL = "https://www.oref.org.il//Shared/Ajax/GetAlarmsHistory.aspx"
```

Comment:
store URLs in .env file

you have an example here:
https://dev.to/jakewitcher/using-env-files-for-environment-variables-in-python-applications-55a1